xml/src/test/scala/akka/stream/alpakka/xml/scaladsl/XmlWritingTest.scala

```diff
@@ -114,6 +114,50 @@ class XmlWritingTest extends WordSpec with Matchers with BeforeAndAfterAll with
       resultFuture.futureValue(Timeout(3.seconds)) should ===(doc)
     }
```

Comment:
Could you add Xml Writer tests with namespace use, please.

Reply:
I have improved the process because of missing cases. I have added few tests. Tell me if I still need to had some.